apps/epic-web/src/pages/events/[event].tsx

```diff
@@ -111,7 +114,7 @@ export type EventPageProps = {
   event: Event
   quantityAvailable: number
   purchaseCount: number
-  product: Product
+  product: SanityProduct
```

Comment:
this is actually expecting SanityProduct down the road :/ there's quite a few dependencies on it, seems like it would be a bigger refactor